combat-trainer.lic

```diff
@@ -2664,6 +2664,10 @@ class AttackProcess
       game_state.action_taken
       ammo = Regexp.last_match(2)
       bput("stow my #{ammo}", 'You pick up', 'Stow what')
+      if game_state.dual_load? && game_state.weapon_skill.eql?('Bows')
```

Comment:
❓  The `dual_load?` function checks that weapon skill is 'Bow', which is different than 'Bows' here (note the plural). Why the difference?

Reply:
Was an error, I just have dual_load? in mine, and I dunno why I added in the skill check, also Bows should be Bow. I just didn't translate it over. Removed altogether.